playbooks/site.yml

```diff
@@ -35,13 +35,180 @@
   - docker
   tags: docker
 
+- hosts: web
+  tasks:
+    - name: stop synapse docker container
+      docker_container:
+        name: "synapse"
+        state: stopped
+
+    - name: create dump of old synapse database
+      shell: docker exec synapse-postgres pg_dump synapse -U synapse >> /root/synapse-dump.sql
```

Comment:
Why is this appeding to the dump file? The step will not be run if the file already exists